templates/c2s-pharmacy-finder/1/rancher-compose.yml

```diff
@@ -0,0 +1,86 @@
+catalog:
+  name: nhsuk_c2s
+  version: 1.0.0
+  description: |
+    Connecting to Services (Using Rancher LB)
+  minimum_rancher_version: v0.59.0
+  questions:
+    - variable: "c2s_docker_image_tag"
+      description: "c2s Docker image tag"
+      label: "c2s Docker image tag:"
+      required: true
+      default: "latest"
+      type: "string"
+    - variable: "nearbyservices_docker_image_tag"
+      description: "nearby-services-api Docker image tag"
+      label: "nearby-services-api Docker image tag:"
+      required: true
+      default: "latest"
+      type: "string"
+    - variable: "mongodb_pharmacy_docker_image_tag"
+      description: "mongodb_pharmacy Docker image tag"
+      label: "nearby-services-api Docker image tag:"

```

Comment:
Label value should be `mongodb_pharmacy Docker image tag:` (assuming the same pattern employed as other questions)